python-flask-example-heroku/config.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    SECRET_KEY = os.getenv("SECRET_KEY", "this-is-read-from-env")
```

Comment:
According to the tutorial, this should be the following:

```suggestion
    SECRET_KEY = os.getenv("SECRET_KEY", "this-is-the-default-key")
```